### PR TITLE
More contact->communication kerfuffles <PD-1523>

### PR DESCRIPTION
**Pull request by tdphillips**
Tests pass.

### CODE REVIEW

File: static/record_overview.167-10.js [hunk 1]
```diff
@@ -327,6 +327,9 @@ function build_data() {
     $(".sidebar > :input:has(option)").each(function() {
         if($(this).val() != 'all') {
             var data_key = $(this).prev('label').html().replace(":", "").replace(" ","_").toLowerCase();
+            if (data_key == 'communication_type') {
```
Comment (tdphillips) [inline static/record_overview.167-10.js]: I said I would do this on the Python side, but this posts to at least two urls at the moment. One location is better than >=2.

**Comment by aspidites** [comment]
What's this look like tackling it from the other direction? That is, translating in python instead of javascript? More or less work? Looks legit otherwise.

**Comment by tdphillips** [comment]
A bit more work. As the comment says, the js posts to at least two different urls. That would mean two places in Python, as opposed to one here.

**Comment by aspidites** [comment]
I see. I think I started typing my comment before you submitted your first one. :+1:  in that case, I agree 1 location is better than one.

**Comment by aspidites** [comment]
Confirmed working.

**Pull request merged**